src/test/java/com/snowflake/kafka/connector/fake/SnowflakeFakeSinkConnector.java

```diff
@@ -0,0 +1,53 @@
+package com.snowflake.kafka.connector.fake;
+
+import com.snowflake.kafka.connector.SnowflakeSinkConnector;
+import com.snowflake.kafka.connector.SnowflakeSinkConnectorConfig;
+import com.snowflake.kafka.connector.Utils;
+import com.snowflake.kafka.connector.internal.KCLogger;
+import org.apache.kafka.common.config.ConfigDef;
+import org.apache.kafka.connect.connector.Task;
+import org.apache.kafka.connect.sink.SinkConnector;
+
+import java.util.*;
```

Comment:
Wildcard imports [are not welcome here](https://github.com/snowflakedb/snowflake-kafka-connector/pull/800#discussion_r1518163589).

Reply:
done